src/capwap_http_api_handler.erl

```diff
@@ -258,11 +265,12 @@ fmt_dp_wtp_stats(true, {RcvdPkts, SendPkts, RcvdBytes, SendBytes,
                {fragment_too_old, ErrFragmentTooOld}]} | Acc];
 fmt_dp_wtp_stats(_, _, Acc) -> Acc.
 
-fmt_dp_wtp_stas(false, {MAC, _RadioId, _BSS, _Stats}) ->
-    [{mac, capwap_tools:format_eui(MAC)}];
-fmt_dp_wtp_stas(true, {MAC, _RadioId, _BSS, Stats}) ->
-    {RcvdPkts, SendPkts, RcvdBytes, SendBytes} = Stats,
-    [{mac, capwap_tools:format_eui(MAC)},
+fmt_dp_wtp_stas(false, STA) ->
+    [{mac, list_to_binary(capwap_tools:format_eui(element(1, STA)))}];
+fmt_dp_wtp_stas(true, STA) ->
+    MAC = element(1, STA),
+    {RcvdPkts, SendPkts, RcvdBytes, SendBytes} = element(size(STA), STA),
```

Comment:
Why the change to the magic tuple matching with the element helper?

Reply:
Because was detected that the tuple can have size 5 instead of expected size 4. When and why tuple size was changed, need to investigate a https://github.com/travelping/capwap-dp. Or - this function not was working correct before for `wtp_stats`

Reply:
the station information should not change in the DP. If it does, it needs to fixed in the DP, not here.

Reply:
this is the STA encoding in capwap-dp: https://github.com/travelping/capwap-dp/blob/master/src/capwap-dp.c#L297-L309

It's a 5 tuple: `{MAC, VLAN, RadioId, BSSId, Stats}`

Last change was 4 years ago.

Reply:
I was update previous commit - I also guess that this mapping did not work properly before , because, maybe the tuple size was always `5`, but due to the fact that I cannot catch all the possible options, I assumed to provide a flexible solution.

Reply:
Got it, thanks, so no make sense to try keep 4 and 5, we need focused only on tuple size 5

Reply:
yes, the 5 tuple is the only supported format.

I guess the HTTP handler was never tested properly.

Reply:
Fixed. Thanks.